### PR TITLE
DOC: '10 minutes to pandas' - <TAB> completion section now show a con…

### LINKED ISSUE #31526: DOC: 10 minutes - <TAB> completion section showing inconsistent suggestions
#### Code Sample, a copy-pastable example if possible

From the very end of the section "[Object Creation](https://pandas.pydata.org/pandas-docs/stable/getting_started/10min.html#object-creation)"
```python

In [12]: df2.<TAB>  # noqa: E225, E999
df2.A                  df2.bool
df2.abs                df2.boxplot
df2.add                df2.C
df2.add_prefix         df2.clip
df2.add_suffix         df2.clip_lower
df2.align              df2.clip_upper
df2.all                df2.columns
df2.any                df2.combine
df2.append             df2.combine_first
df2.apply              df2.consolidate
df2.applymap
df2.D
```
And the following text:
```rst
As you can see, the columns ``A``, ``B``, ``C``, and ``D`` are automatically
tab completed. ``E`` is there as well; the rest of the attributes have been
truncated for brevity.
```

#### Problem description
In the text we read that columns `A`, `B`, `C` and  `D` are shown. However, `df2.B` isn't shown at all

#### Expected Output

I think we could just add some more <TAB> completions, along side `df2.B` to make the documentation more consistent to what is being said.

Something similar to:

```python

In [12]: df2.<TAB>  # noqa: E225, E999
df2.A                  df2.bool
df2.abs                df2.boxplot
df2.add                df2.C
df2.add_prefix         df2.clip
df2.add_suffix         df2.columns
df2.align              df2.combine
df2.all                df2.combine_first
df2.any                df2.D
df2.append             df2.describe
df2.apply              df2.diff
df2.applymap           df2.div
df2.B                  df2.divide
```

##### Note

I'd like to know what does `# noqa: E225, E999` means. I didn't see this reference early in this guide

### PR DESCRIPTION
**Pull request by Cardosaum**
…sistent block of suggestions. Solve issue #31526

- [x] closes #31526
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

I didn't run those listed tests above, however I doubt it will break something.

This PR only correct a *Typo* I found on documentation. I hope I'll be useful to improve this project.

### CODE REVIEW

File: doc/source/getting_started/10min.rst [hunk 1]
```diff
@@ -67,20 +67,20 @@ will be completed:
    @verbatim
    In [1]: df2.<TAB>  # noqa: E225, E999
    df2.A                  df2.bool
-   df2.abs                df2.boxplot
+   df2.abs()              df2.boxplot
```
Comment (gfyoung) [inline doc/source/getting_started/10min.rst]: I can't check this, but are you sure we actually provide parentheses?  That seems inconsistent with the other ones.

**Comment by gfyoung** [comment]
@Cardosaum : First off, welcome to `pandas` 🎉 

I have a quick question about your changes.  Otherwise, would be good if someone else from @pandas-dev/pandas-core could review this, as I can't check this.

File: doc/source/getting_started/10min.rst (hunk 1 above)
Comment (Cardosaum) [reply doc/source/getting_started/10min.rst]: Oh, you are right!

sorry for this mistake.

**Comment by Cardosaum** [comment]
I corrected this error, @gfyoung . 

Thanks for spotting this!

**Comment by jreback** [comment]
thanks @Cardosaum

**Pull request merged**